core/src/main/java/com/orange/ouds/core/component/OudsBadge.kt

```diff
@@ -431,34 +587,98 @@ private fun PreviewOudsBadge(@PreviewParameter(OudsBadgePreviewParameterProvider
 internal fun PreviewOudsBadge(theme: OudsThemeContract, darkThemeEnabled: Boolean, parameter: OudsBadgePreviewParameter) =
     OudsPreview(theme = theme, darkThemeEnabled = darkThemeEnabled) {
         with(parameter) {
-            PreviewEnumEntries<OudsBadgeSize, OudsBadgeStatus> { size, status ->
+            PreviewGrid(
+                columns = enumEntries<OudsBadgeSize>(),
+                rows = statuses,
+                columnTitle = { it.name },
+                rowTitle = { it.name }
+            ) { size, status ->
+                count?.let {
+                    OudsBadge(
+                        count = count,
+                        status = status,
+                        size = size,
+                        enabled = enabled
+                    )
+                }.orElse {
+                    OudsBadge(
+                        status = status,
+                        size = size,
+                        enabled = enabled
+                    )
+                }
+            }
+        }
+    }
+
+internal data class OudsBadgePreviewParameter(
+    val statuses: List<OudsBadgeStatus> = listOf(
```

Comment:
Unlike `OudsBadgeWithIconPreviewParameter`, the `statuses` property here is unused and can be removed.